Source/OIDTokenRequest.m

```diff
@@ -274,7 +275,12 @@ - (NSURLRequest *)URLRequest {
   NSMutableDictionary *httpHeaders = [[NSMutableDictionary alloc] init];
 
   if (_clientSecret) {
-    NSString *credentials = [NSString stringWithFormat:@"%@:%@", _clientID, _clientSecret];
+    // https://tools.ietf.org/html/rfc6749#section-2.3.1
```

Comment:
nit: I'd add some text here like 
// form-urlencode the client id and secret per RFC 6749 Section 2.3.1
// https://tools.ietf.org/html/rfc6749#section-2.3.1